src/components/TextField/TextField.jsx

```diff
@@ -77,10 +77,17 @@ const TextField = React.createClass(createLucidComponentDefinition({
 		onChangeDebounced: func,
 
 		/**
+		 * Fires an event on every keydown
+		 *
+		 * Signature: `(value, { event, props }) => {}`
+		 */
+		onKeyDown: func,
+
+		/**
 		 * Fires an event when the user hits "enter" from the TextField. You
 		 * shouldn't use it if you're using `isMultiLine`.
 		 *
-		 * Signature: `(value, { event, props }) => {}`
+		 * Signature: `(event) => {}`
```

Comment:
I wanted to leave `onKeyDown` vanilla since it wasn't really supposed to be part of the API of this component. I just wanted to make sure it would work as expected if the consumer was trying to `passThrough` a key down handler.